tests/aggregation/test_multi_binary_aggregation.py

```diff
@@ -0,0 +1,49 @@
+"""
+Simple aggregation tests.
+"""
+import pandas as pd
+import pytest
+from pandas.testing import assert_series_equal
+import typing as tp
+
+from crowdkit.aggregation import MultiBinary, MajorityVote, DawidSkene, Wawa, GLAD
+
+
+@pytest.fixture
+def data_toy() -> pd.DataFrame:
```

Comment:
We already have a fixture with a very similar name. Please rename it to reflect that it is for multi-label data.